cachito/workers/tasks/utils.py

```diff
@@ -181,6 +181,23 @@ def get_request_state(request_id):
     return request["state"]
 
 
+def get_request_packages_and_dependencies(request_id: int):
+    """
+    Get the contents of the packages file from the Cachito API.
+
+    :param request_id: the Cachito request ID this is for
+    :raises CachitoError: if the connection fails or the API returns an error response
+    """
+    log.debug("Getting packages file for request %d", request_id)
+    request = _get_request_or_fail(
+        request_id,
+        connect_error_msg=f"Packages file could not be loaded for request {request_id}: {{exc}}",
+        status_error_msg=f"Packages file could not be loaded for request {request_id}: {{exc}}",
```

Comment:
Could be nice to have different messages so that we can differentiate between 404/500 responses from the API vs. straight up connection errors more easily